### ISSUE TITLE
1.0.1 Release of XR and OpenXRVk

### ISSUE THREAD

**Issue by ds5678** [o3de/o3de-extras#667]
#608 increased the version of these gems, but the changes never got published.

**Comment by lumberyard-employee-dm** [comment]
Yes, the link to the zip of the Gem should be updated to point to version 1.0.1 in the [download_source_uri](https://github.com/o3de/o3de-extras/blob/development/Gems/XR/gem.json#L18).

@spham-amzn @galibzon Also the zip for the new release of the Gem needs to be uploaded to github at URL https://github.com/o3de/o3de-extras/releases/download/2.0/xr-1.0.1-gem.zip

**Comment by jhanca-robotecai** [comment]
The newest releases are published correctly, please reopen if you find any problems.
